packages/plugins/rubygem-kubeclient/rubygem-kubeclient.spec

```
@@ -0,0 +1,99 @@
+# Generated from kubeclient-4.3.0.gem by gem2rpm -*- rpm-spec -*-
+# template: scl
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name kubeclient
+
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 4.3.0
+Release: 1%{?dist}
+Summary: A client for Kubernetes REST api
+Group: Development/Languages
+License: MIT
+URL: https://github.com/abonas/kubeclient
+Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
+
+# start specfile generated dependencies
+Requires: %{?scl_prefix_ruby}ruby(release)
+Requires: %{?scl_prefix_ruby}ruby >= 2.2.0
+Requires: %{?scl_prefix_ruby}ruby(rubygems)
+Requires: %{?scl_prefix}rubygem(rest-client) >= 2.0
+Requires: %{?scl_prefix}rubygem(rest-client) < 3
+Requires: %{?scl_prefix}rubygem(recursive-open-struct) >= 1.0
+Requires: %{?scl_prefix}rubygem(recursive-open-struct) < 2
+Requires: %{?scl_prefix}rubygem(recursive-open-struct) >= 1.0.4
+Requires: %{?scl_prefix}rubygem(http) >= 3.0
+Requires: %{?scl_prefix}rubygem(http) < 4
+BuildRequires: %{?scl_prefix_ruby}ruby(release)
+BuildRequires: %{?scl_prefix_ruby}ruby >= 2.2.0
+BuildRequires: %{?scl_prefix_ruby}rubygems-devel
+BuildArch: noarch
+Provides: %{?scl_prefix}rubygem(%{gem_name}) = %{version}
+# end specfile generated dependencies
+
+%description
+A client for Kubernetes REST api.
+
+
+%package doc
+Summary: Documentation for %{pkg_name}
+Group: Documentation
+Requires: %{?scl_prefix}%{pkg_name} = %{version}-%{release}
+BuildArch: noarch
+
+%description doc
+Documentation for %{pkg_name}.
+
+%prep
+%{?scl:scl enable %{scl} - << \EOF}
+gem unpack %{SOURCE0}
+%{?scl:EOF}
+
+%setup -q -D -T -n  %{gem_name}-%{version}
+
+%{?scl:scl enable %{scl} - << \EOF}
+gem spec %{SOURCE0} -l --ruby > %{gem_name}.gemspec
+%{?scl:EOF}
+
+%build
+# Create the gem as gem install only works on a gem file
+%{?scl:scl enable %{scl} - << \EOF}
+gem build %{gem_name}.gemspec
+%{?scl:EOF}
+
+# %%gem_install compiles any C extensions and installs the gem into ./%%gem_dir
+# by default, so that we can move it into the buildroot in %%install
+%{?scl:scl enable %{scl} - << \EOF}
+%gem_install
+%{?scl:EOF}
+
+%install
+mkdir -p %{buildroot}%{gem_dir}
+cp -pa .%{gem_dir}/* \
+        %{buildroot}%{gem_dir}/
+
+%files
+%dir %{gem_instdir}
+%exclude %{gem_instdir}/.gitignore
+%exclude %{gem_instdir}/.rubocop.yml
+%exclude %{gem_instdir}/.travis.yml
+%license %{gem_instdir}/LICENSE.txt
+%{gem_instdir}/RELEASING.md
```

Can you move this to `%files doc`?

```suggestion
%doc %{gem_instdir}/RELEASING.md
```